vendor/github.com/heimweh/go-pagerduty/pagerduty/business_service.go

```diff
@@ -0,0 +1,140 @@
+package pagerduty
+
+import "fmt"
+
+// BusinessServiceService handles the communication with business service
+// related methods of the PagerDuty API.
+type BusinessServiceService service
```

Comment:
Is it to prepare Business service Terraform PagerDuty resource ?
Do you know if it will be available soon ?